dataservice/api/read_group/schemas.py

```diff
@@ -40,15 +31,6 @@ class Meta(BaseSchema.Meta):
     _links = ma.Hyperlinks({
         'self': ma.URLFor(Meta.resource_url, kf_id='<kf_id>'),
         'collection': ma.URLFor(Meta.collection_url),
-        'genomic_files': ma.URLFor('api.genomic_files_list',
-                                   read_group_id='<kf_id>')
+        'read_group_genomic_files': ma.URLFor(
```

Comment:
I think this is a mistake. This link on its own is mostly useless to the API user when what they really want is the list of genomic files.

Reply:
Hmm yea probably should have kept this functionality. I think I thought I couldn't do this for some reason if I reverted back to the "standard" endpoint format.. let me see if i can add this functionality back in while still keeping the /read-group-genomic-files endpoint